examples/dpi/Makefile

```diff
@@ -5,4 +5,6 @@
 PATH_TO_MK = ../../mk
 SUBDIRS = main pattern
 
+export GO_COMPILE_FLAGS += -tags hyperscan_v4
```

Comment:
You should also change leaf.mk:15 to add `gcflags` to `GO_COMPILE_FLAGS` instead of assigning them. Otherwise with NFF_GO_DEBUG this will not work.

Reply:
Done